README.md

```diff
@@ -47,7 +47,7 @@ zmqpubrawtx=tcp://127.0.0.1:29000
 On **__testnet__**, you also need to make sure that all your UTXOs are `p2sh-of-p2wpkh`.
 To do this, use the debug console, create a new address with `getnewaddress`, import it as a witness address with `addwitnessaddress`, and
 send all your balance to this witness address. 
-If you need to create and send funds manually, don't forget to create and specify a witness address for the change output (this option is avaliable on the GUI once you set the `Enable coin control features` wallet option).
+If you need to create and send funds manually, don't forget to create and specify a witness address for the change output (this option is avaliable on the GUI once you set the `Enable coin control features` wallet option). Once you have enabled "coin control features," you must go to the send tab, click on the inputs button and then right click any non-segwit addresses and select "lock unspent." This will allow Eclair to run properly as it does not accept any non-segwit addresses."
```

Comment:
You don't need to select "lock unspent" and I wouldn't say that eclair does not accept non-segwit addresses. We use bitcoind as a wallet and to avoid malleability issues require users to make sure that all their utxos are p2h-of-p2pkh. It's a bit clumsy (you need to manage change addresses manually) and we can only check that addresses are ps2h. This will go away as soon as 0.16 is released (you can already experiment with the RC, just make sure that the address type is `p2sh-segwit` and enable the `addwitnessaddress` RPC call).  Then we'll add support for "real" segwit addresses.